docs/modules/deploy/pages/deploying-on-aws.adoc

```diff
@@ -92,9 +92,108 @@ The following optional properties can be configured[[ec2_properties]]:
 
 Note that if you don't specify any of the properties, then Hazelcast uses the IAM Role assigned to the EC2 Instance to form a cluster from all Hazelcast members running in same region.
 
-=== EC2 Hazelcast Client Configuration
+=== Client Discovery of EC2 Hazelcast Cluster
 
-Hazelcast Client discovery parameters are the same as mentioned above.
+Hazelcast Client can automatically discover the Hazelcast cluster running in EC2 environment.
+
+- *Client running in EC2:*
```

Comment:
Is it possible to fix these titles?
<img width="578" alt="Screenshot 2023-07-17 at 16 58 48" src="https://github.com/hazelcast/hz-docs/assets/6005622/841d639a-0eef-44ae-9b12-76eb8e1137f5">

Reply:
> What do you mean?

As you understand already, converting them into proper titles 🙁 

>  I've tried making them L4 titles (====), but they are not distinguishable from the text, then

hmm, maybe the doc team can help us with it. @Serdaro do you have any solution?

Reply:
I have been using Serge's solution, too (making these supposed-to-be-headings bold, provided that they don't need to be linkables). Let me work with the UI and see.